### ISSUE TITLE
confusing capitalization for querying

### ISSUE THREAD

**Issue by MartinSteffen** [inukshuk/jekyll-scholar#316]
II have the following remark. I have started using jekyll, and with that
also jekyll-scholar, which is a very valuable extension.

It costed be some time to overcome a roadblock with querying my bibtex
"data-base". Perhaps it's explained in the documentation, but it was just
unexpected. It's about capitalization. I am vaguely aware that BibTeX is
case insensitive, for instance for the keys, I know that refkey, REFKEY, and
Refkey are all the same (so I actually avoid capitalization for keys for
that reason). The tool I use maintaining my bibs (emacs), adds entries
capitalization. For instance, the template for PhD theses would look like

@PhDThesis{,
  author	= "",
 [....]
 }

with @PhDThesis using some capitals (other entries analogously). That's suggested by my editor,  so I
leave it like that. I would expect, other editors and bib maintenance
tools also make use of capitals, even if they don't mean anything.

So far so good. When I wanted to /query/ my data base, I used queries like

{% bibliography --query @PhDThesis[ author ^= *somename  ]%}

but there are no matches. If I use phdthesis[author ^= *SomeName ] as
pattern, I get the result I want. The illustrative examples in the
documentations don't use capitalization, but I did not expect that one
/has/ to use lower-case (even if I happen to use capitalization in my
entries).

I would find matching irrespective of capitalization (in the pattern used
for matching as well as the string in the bibfile) more intuitive. or at
least some remark at the examples would be welcome "Note: the query pattern
/has/ to be lower case".

Thanks, Martin

**Comment by inukshuk** [comment]
Good point! Please try updating the `bibtex-ruby` Gem to 5.1.6, this should make the comparison case insensitive.
